_playbook/pdf.md

```diff
@@ -15,12 +15,16 @@ It is possible to create PDF files that meet requirements of Section 508, but it
 
 The UK's Government Digital Services model decreases the reliance on PDFs and is [actively working to move away from them online and pushing for HTML content instead](https://gds.blog.gov.uk/2018/07/16/why-gov-uk-content-should-be-published-in-html-and-not-pdf/). When publishing a PDF is unavoidable, ideally an HTML or MSWord version of the content should be available as well.
 
+For creating accessible documents in Microsoft Word, Excel or Powerpoint, is the preferred method. Microsoft offers an Accessibilty Checker tool, although this may not be an extensive check of the document, it does show major accessibility issues needed to fix.
+
+Visit [Creating Accessible Documents](/documents.md).
```

Comment:
I think this might not work because the page is under the playbook (from your change) so I think it should be `/playbook/documents`